generators/path_generator

```diff
@@ -49,6 +50,10 @@ for cfg in glob.glob(os.path.join(conf_dir, 'generators', 'meta-path',
             for f in glob.glob(os.path.abspath(os.path.join(third_party_dir,
                                                             *path, match))):
 
+                if os.path.splitext(os.path.basename(f))[0] in blacklist:
```

Comment:
Can we change the check to check the full string? `in` will also match substrings, so if there are tests called e.g. `number`, `number-signed`, etc, it will not be possible to blacklist `number-signed` without blacklisting `number`

Reply:
BTW. maybe we could also use the blacklisting with the file extension to avoid similar situations?

Reply:
now it checks the full filename with extension and issue with substrings should be fixed